apps/site/assets/tsconfig.webpack.json

```diff
@@ -2,7 +2,6 @@
   "extends": "./tsconfig",
   "exclude": [
     "**/__tests__/**/*",
-    "node_modules",
     "vendor/",
```

Comment:
I'm curious is this might work instead of removing `node_modules` entirely: https://github.com/webpack/webpack/issues/2031#issuecomment-219040479

Reply:
whoops, that's probably syntax for an older webpack version, maybe the newer comments have a better approach

Reply:
Or with an [include](https://github.com/babel/babel-loader/issues/171#issuecomment-486380160)

Reply:
Good call on the `include`, I'm trying that approach now as it seems more reliable than regexes. I think we do still need to remove this line though, since it would filter out `node_modules` at the ts-loader level even after it's been included at the Webpack level. This is not otherwise a concern because we're also filtering out `node_modules` in the Webpack config everywhere we use ts-loader.